### PR TITLE
3.0, dev: some format fix in DM benchmark report

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? 

some format fix in DM benchmark

### What is the related PR or file link(s)? 
#1859 

### Which version does your change affect? 

3.0, dev

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @yikeke @csuzhangxc

File: dev/benchmark/dm-v1.0-ga.md [hunk 1]
```diff
@@ -111,24 +115,26 @@ sysbench --test=oltp_insert --tables=4 --mysql-host=172.16.4.40 --mysql-port=330
...
+| 4                       | 0.523            | 360.1        | 10.7            | 0.41                 |
+| 8                       | 0.986            | 267.0        | 14.5            | 0.93                 |
+| 16                      | 2.022            | 265.9        | 14.5            | 2.68                 |
+| 32                      | 3.778            | 262.3        | 14.7            | 6.39                 |
+| 64                      | 7.452            | 281.9        | 13.7            | 8.00                 |
 
 #### 导入数据时每条插入语句包含行数不同的情况下的性能测试对比
 
-该测试中全量导入的数据量为 3.78GB，load 处理单元 `pool-size` 大小为32. 插入语句包含行数通过 mydumper 的 `--statement-size` 来控制。
+该测试中全量导入的数据量为 3.78 GB，load 处理单元 `pool-size` 大小为 32. 插入语句包含行数通过 mydumper 的 `--statement-size` 来控制。
```
Comment (yikeke) [inline dev/benchmark/dm-v1.0-ga.md]:
```suggestion
该测试中全量导入的数据量为 3.78 GB，load 处理单元 `pool-size` 大小为 32。插入语句包含行数通过 mydumper 的 `--statement-size` 来控制。
```

File: v3.0/benchmark/dm-v1.0-ga.md [hunk 2]
```diff
@@ -111,24 +115,26 @@ sysbench --test=oltp_insert --tables=4 --mysql-host=172.16.4.40 --mysql-port=330
...
+| 4                       | 0.523            | 360.1        | 10.7            | 0.41                 |
+| 8                       | 0.986            | 267.0        | 14.5            | 0.93                 |
+| 16                      | 2.022            | 265.9        | 14.5            | 2.68                 |
+| 32                      | 3.778            | 262.3        | 14.7            | 6.39                 |
+| 64                      | 7.452            | 281.9        | 13.7            | 8.00                 |
 
 #### 导入数据时每条插入语句包含行数不同的情况下的性能测试对比
 
-该测试中全量导入的数据量为 3.78GB，load 处理单元 `pool-size` 大小为32. 插入语句包含行数通过 mydumper 的 `--statement-size` 来控制。
+该测试中全量导入的数据量为 3.78 GB，load 处理单元 `pool-size` 大小为 32. 插入语句包含行数通过 mydumper 的 `--statement-size` 来控制。
```
Comment (yikeke) [inline v3.0/benchmark/dm-v1.0-ga.md]:
```suggestion
该测试中全量导入的数据量为 3.78 GB，load 处理单元 `pool-size` 大小为 32。插入语句包含行数通过 mydumper 的 `--statement-size` 来控制。
```

**Comment by amyangfei** [comment]
thanks, PTAL @yikeke

**Pull request merged**